Documentation/gettingstarted/k8s-install-gke-validate.rst

```diff
@@ -6,19 +6,23 @@ You can monitor as Cilium and all required components are being installed:
 .. parsed-literal::
 
     kubectl -n cilium get pods --watch
-    NAME                                    READY   STATUS              RESTARTS   AGE
-    cilium-operator-cb4578bc5-q52qk         0/1     Pending             0          8s
-    cilium-s8w5m                            0/1     PodInitializing     0          7s
-    coredns-86c58d9df4-4g7dd                0/1     ContainerCreating   0          8m57s
-    coredns-86c58d9df4-4l6b2                0/1     ContainerCreating   0          8m57s
+    NAME                               READY   STATUS            RESTARTS   AGE
```

Comment:
might be slightly more succinct to use something like:
```
kubectl wait --for=condition=Ready -n kube-system --all pod
```

Reply:
That's a nice command, I didn't know about it! I think that there are several places in the doc where the `get pods --watch` command is used so for consistency I will leave it as is. However, it may be an improvement to update this across the whole documentation in another PR.